app/controllers/api/v1/me_controller.rb

```diff
@@ -4,13 +4,27 @@ class API::V1::MeController < ApplicationController
     
     def show
         render json: User.find(doorkeeper_token.resource_owner_id)
-            .as_json(include: [
-                organisation: { 
-                    only: [:id, :name], 
-                    include: [
-                        services: {only: [:id, :name]}
-                    ]
+            .as_json(include: {
+                organisation: {
+                  only: [:id, :name],
+                  include: {
+                    services: {
+                      only: [:id, :name],
+                      include: {
+                        service_at_locations: {
+                          only: [:id],
+                          include: {
+                            location: {
+                              only: [:id, :name, :address_1, :city, :postal_code]
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
                 }
-            ])
+              })
```

Comment:
I think this needs to be abstracted out into some new endpoints:

- `api/v1/organisations`
- `api/v1/organisations/:id`

- `api/v1/users/`
- `api/v1/users/:id`
- `api/v1/users/:id/organisations`
- `api/v1/users/:id/services`

- `api/v1/me`
- `api/v1/me/services`
- `api/v1/me/organisations` <-- we would only need to create this one for now